xcmailr-load-test-suite/pom.xml

```diff
@@ -16,12 +16,21 @@
     <description>XCMailr load test suite using XLT</description>
 
     <properties>
+        <java.version>21</java.version>
         <license.skipFormat>false</license.skipFormat>
         <maven.deploy.skip>true</maven.deploy.skip>
     </properties>
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.11.0</version>
+                <configuration>
+                    <release>${java.version}</release>
+                </configuration>
+            </plugin>
```

Comment:
Please revert this change as upgrading to Java21 is a separate issue and not related to adding support for secondary SMTP.

Reply:
Done.